.github/workflows/scanner-versioned-definitions-update.yaml

```
@@ -223,23 +232,35 @@ jobs:
       uses: ./.github/actions/download-artifact-with-retry
       with:
         name: nvd
-        path: nvd.zip
+        path: .
 
     - name: Build updater
       run: |
         echo "Building updater for pull request ${{ env.PR_NAME }}..."
         make tag
         make -C scanner bin/updater
 
+    - name: Sanity check NVD zip
+      run: |
+        if [ ! -f ./nvd.zip ]; then
+          echo "nvd.zip not found!"
+          exit 1
+        fi
+
     - name: Create bundle output directory
       run: mkdir -p definitions/${{ github.event.pull_request.number }}
 
     - name: Run Updater (single bundle)
       run: |
+        # Use fully qualified path to refer to the NVD bundle
+        STACKROX_NVD_ZIP_PATH="$(pwd)/nvd.zip"
+        export STACKROX_NVD_ZIP_PATH
```

Nipick: OK if we need the absolute path for some reason, but if not, I think it's easier to read if we use `env.STACKROX_NVD_ZIP_PATH: "nvd.zip"`.
```suggestion
        # Use fully qualified path to refer to the NVD bundle
        export STACKROX_NVD_ZIP_PATH="$(pwd)/nvd.zip"
```

---

Unfortunate this change cannot pass Lint check

---

Why?

---

```
run: |
        # Use fully qualified path to refer to the NVD bundle
        export STACKROX_NVD_ZIP_PATH="$(pwd)/nvd.zip"
        scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" "definitions/${{ github.event.pull_request.number }}"

shellcheck reported issue in this script: SC2155:warning:2:8: Declare and assign separately to avoid masking return values [shellcheck]
    |
254 |       run: |
    |       ^~~~
```

---

I think it may actually be complaining about the export combined with the $(pwd) and thinking cannot effectively capture the exit code (just referencing the wrong line in the lint output). Use `$PWD` then:

```
export STACKROX_NVD_ZIP_PATH="$PWD/nvd.zip"
```

But, you might not need this based on the other conversation. Just setting using `env.STACKROX_NVD_ZIP_PATH` would work.

---

Bash ignores return values of commands substitute in qualified variables (`local`, `export`). Use `$PWD` then in this case would work because it is set by the shell. But yeah, whatever.

We might not need this since the relative path is fine.

---

Thanks for showing me the reason, learned something. :+1: